_scalingo_api/collaborators.md

```diff
@@ -198,3 +198,87 @@ curl -H "Accept: application/json" -H "Content-Type: application/json" \
 ```
 
 Returns 204 No Content
+
+--- row ---
+
+# ExtendedCollaborators
```

Comment:
Do we call it `ExtendedCollaborators`? In scalingo.js, we end up just calling it `Collaborators`

Reply:
Yeah i think you're right. Also I think we need to wait until we decided what to do [here](https://scalingo.slack.com/archives/C04TEKMR7FY/p1732724257848669)